accepted/2023/Add-dotnet-search.md

```diff
@@ -113,13 +113,13 @@ Here are the details of each verbosity level:
 - **`--verbosity normal`** : The output will provide a moderate amount of information. For each package, it will display the following metadata:
   - Package ID
   - Latest Version
-  - Authors
+  - Owners
   - Total Downloads
   
 - **`--verbosity detailed`** : The output will be highly detailed. For each package, it will display the following metadata:
   - Package ID
   - Latest Version
-  - Authors
+  - Owners
```

Comment:
Sounds good to show Owner for this first iteration, and if we learn that customers want Author, we can discuss a second column for that in the next iteration.